engine/runner/src/main/java/org/enso/runner/Main.java

```diff
@@ -916,7 +916,9 @@ private void runMain(
         for (var e : parsedArgs) {
           listOfArgs = join(e, listOfArgs);
         }
-        var res = main.execute(listOfArgs.reverse());
+        logger.debug("Executing the main function with arguments {}", listOfArgs.mkString(", "));
+        listOfArgs = listOfArgs.reverse();
+        var res = main.execute(listOfArgs);
```

Comment:
We execute the function with reversed `listOfArgs`. The log will show arguments backward, which can cause confusion.

Reply:
Right thanks for pointing out! I totally meant to have the log line _after_ reversing...